### PR TITLE
Add workflow to run multiple WGSFromBam in parallel

### PR DESCRIPTION
**Pull request by vladsavelyev**
Add a workflow to run WGSFromBam.wdl (a BAM to gVCF workflow - our modified version of the Broad's one) on multiple samples in parallel.

Going to also push the code into https://github.com/populationgenomics/cpg-fewgenomes that generates the input for this workflow.

(Also, adds a tiny fix WGSFromBam.wdl itself).

### CODE REVIEW

File: pipelines/broad/dna_seq/germline/single_sample/wgs/WGSMultipleSamplesFromBam.wdl [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+
+workflow WGSMultipleSamplesFromBam {
+
+  String pipeline_version = "1.0.0"
+
+  input {
+    File sample_map
+
+    DNASeqSingleSampleReferences references
```
Comment (illusional) [inline pipelines/broad/dna_seq/germline/single_sample/wgs/WGSMultipleSamplesFromBam.wdl:13]: Don't you need to import these structs?

Comment (illusional) [reply pipelines/broad/dna_seq/germline/single_sample/wgs/WGSMultipleSamplesFromBam.wdl:13]: I guess the equivalent `*SamplesFromFastqs.wdl` doesn't, so this is fine.

**Review by illusional — APPROVED** [review]
Your WDL looks good to me but I do get when I do get an error when run womtool / miniwdl over your new WDL file (but the error is in `pipelines/broad/dna_seq/germline/single_sample/wgs/WGSFromFastq.wdl`:

```
Failed to import 'WGSFromBam.wdl' (reason 1 of 1): Failed to import 'WGSFromFastq.wdl' (reason 1 of 1): Failed to process workflow definition 'WGSFromFastq' (reason 1 of 1): Failed to process 'call ToBam.FastqsToAlignedBam' (reason 1 of 1): To be called as a sub-workflow it must declare and pass-through the following values via workflow inputs: ApplyBQSR.input_bam_index, BaseRecalibrator.input_bam_index
```

Not your fault, and that step actually does pass all the required inputs, but it's a crappy byproduct of a workflow not having to provide all inputs to a step / tool, because the inputs json could do that.

**Comment by vladsavelyev** [comment]
Look like that was an artifact of merging with the upstream. Pushed a few fixes, it's working without those errors for me now.

Thank Michael for reviewing, and for checking the workflow with womtool and miniwdl!

File: pipelines/broad/dna_seq/germline/single_sample/wgs/WGSMultipleSamplesFromBam.wdl (hunk 1 above)
Comment (vladsavelyev) [reply pipelines/broad/dna_seq/germline/single_sample/wgs/WGSMultipleSamplesFromBam.wdl:13]: Yep, it gets imported implicitly with `import "WGSFromBam.wdl" as WGSFromBam`

**Pull request merged**